pkg/annotations/annotations.go

```diff
@@ -0,0 +1,19 @@
+package annotations
+
+// ContainerType values
+const (
+	// ContainerTypeSandbox represents a pod sandbox container
+	ContainerTypeSandbox = "sandbox"
+
+	// ContainerTypeContainer represents a container running within a pod
+	ContainerTypeContainer = "container"
+
+	// ContainerType is the container type (sandbox or container) annotation
+	ContainerType = "io.kubernetes.cri-containerd.ContainerType"
```

Comment:
How about cri.containerType, cri.sandboxID, cri.sandboxName.
Might want to substitute sandbox for pod, since these labels might get printed out and most folks won't know that sandbox is the internal name for pod...

Reply:
or may be io.kubernetes.containerd.ContainerType and so on below ?

Reply:
@jcvenegas Same question here. Are we going to have different annotation key for cri-o, cri-containerd or dockershim? Shouldn't that be a standard name?

Reply:
@mikebrow @abhi :  "cri.containerType" sounds good to me. It is a more agnostic cri implementation.
About ` podID` or `sanboxID`. What about `PodSandbox` tight to cri api definitions and includes both `pod` and `sandbox`  to avoid confusions.

Reply:
Today kata is looking for different `annotations`. I like the idea  to go beyond  and take a first step towards standardization.